main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/marcodeltorob/golang-bootcamp-2020/config"
+	"github.com/marcodeltorob/golang-bootcamp-2020/controller"
+	"github.com/marcodeltorob/golang-bootcamp-2020/router"
+	"github.com/marcodeltorob/golang-bootcamp-2020/usecase"
+	"github.com/unrolled/render"
+)
+
+func main() {
+	configuration := config.GetConfig()
```

Comment:
Your definition of `config.GetConfig()` has a string var-arg argument, but you are not passing any value to it. Is it expected?